ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/interface_ip_nat_keys.schema.yml

```
@@ -0,0 +1,119 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+$defs:
+  interface_ip_nat_static:
+    type: list
+    primary_key: original_ip
+    items:
+      type: dict
+      keys:
+        access_list:
+          type: str
+          description: "'access_list' and 'group' are mutual exclusive"
+        comment:
+          type: str
+        direction:
+          type: str
+          valid_values: [egress, ingress]
+          description: |-
```

We don't require the "-" in our schema.
```suggestion
          description: |
```

---

Fixed in https://github.com/aristanetworks/ansible-avd/pull/2750/commits/11feb2f39043fcbee097a7d5b7caa793e9b85ed4